cosmos/settings.py

```diff
@@ -275,6 +293,9 @@
 # Celery
 CELERY_BROKER_URL = "amqp://guest:guest@localhost//"
 CELERY_RESULT_BACKEND = "django-db"
+if secret_settings.secrets["CELERY"]["USE_REDIS"]:
+    CELERY_BROKER_URL = secret_settings.secrets["CELERY"]["REDIS_URL"]
+    CELERY_RESULT_BACKEND = secret_settings.secrets["CELERY"]["REDIS_URL"]
```

Comment:
I dont know if we want to set the celery result backend to redis instead of django. Currently we use an addon to django which stores the results and thus also makes them visible from the admin panel, giving a bit more insight if a task succeeded or not. If we do want to switch it to redis, we probably want to remove that plugin as well.